src/Apps/Order2/Routes/Checkout/Components/Order2ReviewStep.tsx

```diff
@@ -122,6 +144,8 @@ export const Order2ReviewStep: React.FC<Order2ReviewStepProps> = ({
   )
 }
 
+export const Order2ReviewStep = injectDialog(Order2ReviewStepComponent)
```

Comment:
non-blocking: I know our old orders app used `injectDialog()` but I've been wondering if there is a more modern pattern somewhere - the old higher-order component that just adds a prop pattern has just evolved in many other places. I wouldn't hold this PR up for that because we can switch it pretty easily though, and I've never looked into it at all.

For reference, [the old orders app dialog implementation is here](https://github.com/artsy/force/blob/339c961be48adf52925fcfb3aaf0586f159aaa19/src/Apps/Order/Dialogs.tsx#L31-L32). The class components, dependency on [unstated](https://www.npmjs.com/package/unstated), etc... just feels old and worth a second look at some point.

Reply:
Edit: It looks like this is the only use of `unstated` in the app. Still ok with merging though.